drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/widget/table/GuidedDecisionTablePresenter.java

```diff
@@ -787,11 +788,19 @@ public void newAttributeOrMetaDataColumn() {
         if ( isReadOnly() ) {
             return;
         }
-        view.newAttributeOrMetaDataColumn();
+        view.newAttributeOrMetaDataColumn( getReservedAttributeNames() );
```

Comment:
Shouldn't be this `view.newAttributeOrMetaDataColumn( getExistingAttributeNames() );`?

I am asking just because of the signature: `newAttributeOrMetaDataColumn( final Set<String> existingAttributeNames );` in: `GuidedDecisionTableView.java.`